common/spec/dependabot/dependency_group_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "dependabot/dependency_group"
+
+RSpec.describe Dependabot::DependencyGroup do
+  describe "#name" do
+    it "returns the name" do
+      my_dependency_group_name = "Darren from work"
```

Comment:
```suggestion
      my_dependency_group_name = "darren-from-work"
```

First, 😂 

Secondly, we aren't enforcing validation yet, but we should aim for names to be snake-cased so they can be used in branch names, etc.